sdk/keyvault/keyvault-keys/test/crypto.test.ts

```diff
@@ -83,6 +84,10 @@ describe("CryptographyClient (all decrypts happen remotely)", () => {
 
   it("sign and verify with RS256", async function() {
     recorder.skip("browser", "Local encryption is only supported in NodeJS");
```

Comment:
If something is not supported in browsers, keep it as a node test.
Keep a separate folder which is exclusive for node tests.
`test/node/*.ts` - node tests
`test/browser/*.ts` - browser tests
`test/*.ts` - common tests for both node and browser

Reply:
I agree with you! We'll revisit how to organize test files later though.

Reply:
Update rollup and the commands in package.json accordingly.

Reply:
I won't be moving things around in this PR.

Reply:
In that case, mention why the test is being skipped in a comment above.

Reply:
Sure!